src/content/en/2020/capabilities.md

```diff
@@ -0,0 +1,335 @@
+  )
+}}
+
+The Async Clipboard API is comparatively new, so its usage is currently rather low. In March 2020, Safari added support for the Async Clipboard API in Safari 13.1. Over the course of 2020, the usage of the `read()` API was growing. In October 2020, the API was called during 0.0003 percent of all page loads in Google Chrome.
+
+### Write Access
+
+Apart from reading operations, the Async Clipboard API also offers two methods for writing content to the clipboard. Again, there's a shorthand method for plain text, called `navigator.clipboard.writeText()`, and one for arbitrary data called `navigator.clipboard.write()`. In Chromium-based browsers, writing to the clipboard while the tab is active does not require permission. Trying to write to the clipboard when the website is in the background does, however. As this method requires a user gesture and permission first, it's not covered by the HTTP Archive metrics. In contrast to the `read()` method, the `write()` method shows an exponential growth in usage, being part of 0.0006 percent of all page loads in October 2020.
+
+The Raw Clipboard Access API ([related Chromium bug](https://bugs.chromium.org/p/chromium/issues/detail?id=897289)), another Fugu capability, might even further enhance the Async Clipboard API by allowing arbitrary data to be copied from or pasted to the clipboard.
+
+## StorageManager API
+
+Web browsers allow users to store data on the user's system in different ways, such as Cookies, the Indexed Database (IndexedDB), the Service Worker's Cache Storage, or Web Storage (Local Storage, Session Storage). In modern browsers, developers can easily [store hundreds of megabytes and even more](https://web.dev/storage-for-the-web/), depending on the browser. When browsers run out of space, they can clear data until the system is no longer over the limit, which can lead to data loss.
+
+Thanks to the [StorageManager API](https://developer.mozilla.org/en-US/docs/Web/API/StorageManager) (part of the [WHATWG Storage Living Standard](https://storage.spec.whatwg.org/#storagemanager)), browsers no longer behave like a black box in that regard: This API allows developers to estimate the remaining space available, and to opt-in to [persistent storage](https://web.dev/persistent-storage/), meaning that the browser will not clear a website's data when disk space is low. Therefore, the API introduces a new `StorageManager` interface on the `navigator` object, currently available on Chrome, Edge, and Firefox.
+
+### Estimate the available storage
+
+Developers can estimate the available storage by calling `navigator.storage.estimate()`. This method returns a promise resolving to an object with two properties: `usage` shows the number of bytes used by the application and `quota` contains the maximum number of bytes available.
+
+{{ figure_markup(
+  image="storage_manager_api_estimate.png",
+  alt="Percent of top pages using the estimate method of the StorageManager API"
```

Comment:
Missing a comma here and in a couple of other places like this, which is causing the test failures. Could you correct?

```suggestion
  alt="Percent of top pages using the estimate method of the StorageManager API",
```

Reply:
@christianliebel I was testing locally and went ahead and committed these fixes